queries/__init__.py

```diff
@@ -253,6 +253,546 @@ def convert(m):
     return re.sub(r"(\d+)", convert, s)
 
 
+_SUB_20_NEUTRAL = {
```

Comment:
Add type annotation: `_SUB_20_NEUTRAL: Mapping[int, str] = {`  (Mapping is a read-only dict)

Reply:
Fixed!